src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,44 @@
+databaseChangeLog:
```

Comment:
This changelog should be in `changes` folder
in db.changelog-master.yaml you should have like 
```
databaseChangeLog:
  - include:
      file: db/changelog/changes/01-create-books-table.yaml
```